docs/Making-a-release.md

```diff
@@ -0,0 +1,11 @@
+# Making a Release of Simulation Construction Set 2
+1. Bump the version in `group.gradle.properties`
+2. Commit only `group.gradle.properties` with a message in the format: "`:bookmark: <version>`"
+3. Create a tag with the version name
```

Comment:
Is this gonna be the `<version>`?

Reply:
Yes

Reply:
Historically the tag names are different than that version names